config.example.wikimedia.yaml

```diff
@@ -17,27 +17,27 @@ default_project: &default_project
             - name: default.group.local
               domains: /./
         parsoid:
-          host: http://parsoid-beta.wmflabs.org
+          host: https://parsoid-beta.wmflabs.org
           grace_ttl: 1
         action:
           apiUriTemplate: "{{'https://{domain}/w/api.php'}}"
           baseUriTemplate: "{{'https://{domain}/api/rest_v1'}}"
         graphoid:
-          host: http://graphoid.wikimedia.org
+          host: https://graphoid.wikimedia.org
```

Comment:
Not routable?

Reply:
Indeed, this doesn't exist any more. Let's change this to `graphoid-beta.wmflabs.org`